site/css/_divider.css

```diff
@@ -6,9 +6,10 @@
 }
 
 .divider:before {
+  padding-left: 2px;
   content:"";
   position: absolute;
-  right: -6px;
+  right: -4px;
```

Comment:
Doesn't affect any other dividers since we only use full width divs. This only lines up the divider with a half width div.